Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/SignContract/Signatures/Exceptions/SignatureNotValidException.cs

```diff
@@ -0,0 +1,3 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Core.SignContract.Signatures.Exceptions;
+
+public sealed class SignatureNotValidException(string signatureText) : InvalidOperationException($"Signature text: '{signatureText}' contains invalid characters.");
```

Comment:
Here is a good example - you have SignatureNotValidException while having "signatureText" as a parameter. IMO Text suffix is not needed

Reply:
Yes, we can get rid of suffix